aws/table_aws_batch_job.go

```diff
@@ -0,0 +1,331 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/batch"
+
+	batchv1 "github.com/aws/aws-sdk-go/service/batch"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsBatchJob(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_batch_job",
+		Description: "AWS Batch Job",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("job_name"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"InvalidParameterValueException"}),
+			},
+			Hydrate: getBatchJob,
+			Tags:    map[string]string{"service": "batch", "action": "DescribeJobs"},
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listAwsBatchJobs,
+			Tags:    map[string]string{"service": "batch", "action": "ListJobs"},
+			KeyColumns: []*plugin.KeyColumn{
```

Comment:
Could you please add the columns job_name, job_definition, and created_at as optional quals? The filter input param supports these values for filtering out the result.

created_at optional column should support the operator(=, <, >, <=, >=), based on the operator we can use the filter input `BEFORE_CREATED_AT` and `AFTER_CREATED_AT`

Note: Please update the table doc for the statement `Only one filter can be used at a time. When the filter is used, jobStatus is ignored`. I am not sure though how the API behaves.